code/script.js

```diff
@@ -1,325 +1,452 @@
+    eyes: "covered",
+    accessories: ["glasses", "hat"],
+    other: [],
   },
   {
-    name: 'Josephine',
-    img: 'images/josephine.svg',
-    hair: 'grey',
-    eyes: 'brown',
+    name: "Josephine",
+    img: "images/josephine.svg",
+    hair: "grey",
+    eyes: "brown",
     accessories: [],
-    other: []
+    other: [],
   },
   {
-    name: 'Josh',
-    img: 'images/josh.svg',
-    hair: 'yellow',
-    eyes: 'green',
+    name: "Josh",
+    img: "images/josh.svg",
+    hair: "yellow",
+    eyes: "green",
     accessories: [],
-    other: []
+    other: [],
   },
   {
-    name: 'Jude',
-    img: 'images/jude.svg',
-    hair: 'black',
-    eyes: 'green',
+    name: "Jude",
+    img: "images/jude.svg",
+    hair: "black",
+    eyes: "green",
     accessories: [],
-    other: []
+    other: [],
   },
   {
-    name: 'Julie',
-    img: 'images/julie.svg',
-    hair: 'black',
-    eyes: 'brown',
-    accessories: ['glasses', 'hat'],
-    other: []
+    name: "Julie",
+    img: "images/julie.svg",
+    hair: "black",
+    eyes: "brown",
+    accessories: ["glasses", "hat"],
+    other: [],
   },
-]
+];
 
 // Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+let secret;
+let currentQuestion;
+let charactersInPlay;
+let numberOfGuesses = 0;
 
 // Draw the game board
 const generateBoard = () => {
-  board.innerHTML = ''
+  board.innerHTML = "";
   charactersInPlay.forEach((person) => {
     board.innerHTML += `
       <div class="card">
         <p>${person.name}</p>
         <img src=${person.img} alt=${person.name}>
         <div class="guess">
           <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          <button class="filled-guess-button small" onclick="guess('${person.name}')">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
+};
+
+/* I thought getting popup alerts with every guess was annoying, 
+so I made a container in the HTML for a log, styled it in CSS,
+and used this addToLog-function instead of alert() to put the messages there instead */
+
+const addToLog = (message) => {
+  log.innerHTML += `<p class="log-entry">${message}</p>`;
+  logWrapper.scrollTop = -log.scrollHeight;
+  // above function will keep the scrollbar scrolled UP because we want the newest message to be at the top
+  // please note the negative sign!
+};
+
+const updateGuessCounter = () => {
+  guessCounter.innerHTML = `Guesses: ${parseInt(numberOfGuesses)}`; // converting the int numberOfGuesses to a string. not sure this is needed actually?
+};
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret =
+    charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+  console.log("the person is set to " + secret.name);
```

Comment:
Maybe remove the console.log on line 259 since it reveals the secret person in the console :-)